CONTRIBUTING.md

```diff
@@ -69,22 +69,21 @@ charmcraft pack
 ### Deploy
 
 OpenSearch has a set of system requirements to correctly function, you can find the list [here](https://opensearch.org/docs/2.6/opensearch/install/important-settings/).
```

Comment:
Worthy changing this to: https://opensearch.org/docs/latest/install-and-configure/install-opensearch/index/